Dockerfiles/mariadb/mariadb.conf

```diff
@@ -0,0 +1,17 @@
+[mariadb]
+# File Key Management
+plugin_load_add = file_key_management
+file_key_management_filename = /etc/mysql/encrypt/keys.enc
+file_key_management_filekey = FILE:/etc/mysql/encrypt/pass.key
+file_key_management_encryption_algorithm = AES_CTR
+
+# InnoDB/XtraDB Encryption Setup
+innodb_default_encryption_key_id = 1
```

Comment:
This is already set to 1 in defaults.
[https://mariadb.com/kb/en/innodb-system-variables/#innodb_default_encryption_key_id] 
Why do we have this defined here?
You have created 100 keys and it is always being set to use the first key. I couldn't understand everything yet, but reading documentation. Is this for development environment?

Reply:
It is for dev. That specific setting is a remnant of my testing and should be removed.